be1-go/hub/organizer.go

```diff
@@ -495,7 +510,15 @@ func (c *laoChannel) processMessageObject(public message.PublicKey, data message
 		}
 
 		c.inboxMu.Lock()
-		msg := c.inbox[msgEncoded]
+		msg, ok := c.inbox[msgEncoded]
+		if !ok {
+			// We received a witness signature before the message itself.
```

Comment:
add some sort of TODO / FIXME label here, so that this issue can be easily found.
I'd also consider creating an issue in Github, to facilitate discovery.